react/features/base/logging/reducer.ts

```diff
@@ -38,9 +37,26 @@ if (navigator.product === 'ReactNative') {
     };
 }
 
+type LogLevel = 'trace' | 'log' | 'info' | 'warn' | 'error';
+
+export interface ILoggingState {
+    config: {
+        defaultLogLevel: LogLevel;
+        disableLogCollector?: boolean;
+        'modules/RTC/TraceablePeerConnection.js': LogLevel;
```

Comment:
Is there a way to type this more generically? The list of strings there is unbounded. We need some kind of `[key: string]: LogLevel`